scripts/get-version-marker.js

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
Frankly, this could be named `version-marker.js`... I think the `get` would make me expect I gave a `getVersionMarker()` function in there, which ain't the case.

But that's not... important.

Reply:
I wanted it to aligned with `get-version` and I didn't want just `version'... Whatever... Not that important...